pages/alert.html

```diff
@@ -1,136 +1,151 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.2/css/all.min.css"/>
-    <link rel="stylesheet" href="/styles.css"/>
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.2/css/all.min.css"
+    />
+    <link rel="stylesheet" href="/styles.css" />
     <title>Dazzle : Alert</title>
-</head>
-<body>
+  </head>
+  <body>
     <div class="grid-layout">
+      <nav></nav>
+      <sidebar class="sidebar scrollbar"></sidebar>
 
-        <nav></nav>
-        <sidebar class="sidebar scrollbar"></sidebar>
+      <div class="main">
+        <div class="component-heading">
+          <h2>Alerts</h2>
+          <p>
+            Alerts are used to attract user's attention for important
+            information without interrupting the user's flow.
+          </p>
+          <p></p>
```

Comment:
I think ```<p>``` tag is unwanted.

Reply:
Yes, it is.  Did not see that. Thanks for pointing it out.